app/src/main/res/values/strings.xml

```diff
@@ -423,7 +423,7 @@
     <string name="kiosk">Kiosk</string>
     <string name="trending">Trending</string>
     <string name="top_50">Top 50</string>
-    <string name="new_and_hot">New &amp; hot</string>
+    <string name="new_and_hot">New and hot</string>
```

Comment:
That's SoundCloud's original name / spelling. We should not change it.

Reply:
Because it is not how an ampersand works.

Reply:
Could you explain that further?

Reply:
That's how my phone settings look like:
<img src="https://user-images.githubusercontent.com/17365767/94336244-df760680-ffe1-11ea-931b-16d3da2b7558.png" width="300" />

Reply:
@opusforlife2 
If you look at what the categories are in "Settings" for Android, it will say "and", not "&", between shared categories. Much like it says "and" for mentioning anything above one thing in NewPipe already.
If needing to specifically make it two distinct things, 
"Option1, option 2" instead of 
"Option1 option 2" is the way to go.

It makes it into languages where it doesn't belong at all 
https://hosted.weblate.org/translate/newpipe/strings/nb_NO/?checksum=35c864b620356a41 https://hosted.weblate.org/translate/newpipe/strings/nb_NO/?checksum=b402d2ded65c5cd6 https://hosted.weblate.org/translate/newpipe/strings/nb_NO/?checksum=8260572532e87eb2

Reply:
Oh. So it's causing problems for translations. Makes sense to change it then. I asked because it's fine grammar-wise.

Reply:
@opusforlife2 I tried to Wolfram Alpha an answer, but didn't work. However "and" is unproblematic.
Also consistency is better to every mention of "and", but specifically
<string name="charset_letters_and_digits">Letters and digits</string> was and remains "and", where it is exactly the same convention as "&" is for the changed strings.

Reply:
I don't think there are any strict rules around it. From what I know, you generally use 'and' everywhere, and only use '&' when you have less space available.